upload-coverage/action.yml

```diff
@@ -37,15 +37,18 @@ runs:
         fi
         # Generate the coverage report.
         lcov --no-external --gcov-tool gcov ${dirs_opts} --capture --output-file lcov.info;
-        # Download the coverage export script.
-        curl --silent --output codecov --fail https://codecov.io/bash || \
-          curl -s -o codecov -f https://codecov.io/bash;
-        VERSION=$(grep -o 'VERSION=\"[0-9\.]*\"' codecov | cut -d'"' -f2);
-        for i in 1 256 512
-        do
-          shasum -a $i -c <(curl -s "https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA${i}SUM" | grep codecov) || \
-          shasum -a $i -c <(curl -s "https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA${i}SUM" | grep codecov)
-        done
-        # Effectively export the coverage report.
-        bash codecov -n "${{ inputs.name }}" -X gcov;
```

Comment:
I don't think the new uploader supports `-X gcov` — do we need to do anything about this?

Reply:
I see the `-X FEATURE` argument in the documentation: https://docs.codecov.com/docs/codecov-uploader#uploader-command-line-arguments but I'm not sure the meaning is the same.

Reply:
I tried grepping `gcov` in the repo of the source code for their new uploader, and it didn't look they support this option.

Reply:
Based on the description, `-X gcov` is supposed to "Disable uploading the file" gcov, if I understand it correctly. Maybe the pattern `-X FEATURE` is just inaccurate and `FEATURE` is just `FILE`/`DIRECTORY`?

Reply:
In the old uploader, `-X gcov` would disable running `gcov` (otherwise I think the uploader would run `gcov` in some circumstances).

In the new uploader, I think only `-X network` is supported. I don't think it does anything with `gcov`, so it's probably fine without the flag.

Reply:
You are probably right. In the description of the bash uploader (https://docs.codecov.com/docs/about-the-codecov-bash-uploader), the case of `-X gcov` is explicitly listed along with other possible values for `FEATURE`, so it looks like the new uploader supports just `FEATURE=network`.